src/consensus/params.h

```diff
@@ -48,6 +48,7 @@ struct Params {
     int nBudgetProposalEstablishingTime; // in seconds
     int nSuperblockStartBlock;
     int nSuperblockCycle; // in blocks
+    int nMinGovernanceQuorum; // Min absolute vote count to trigger an action 
```

Comment:
The pattern isn't very clear imo because there usually aren't many parameters for a given class.  However if there are additional governance parameters I guess it would be better to have them all start with governance.  Will change.